content/en/docs/concepts/scheduling/kube-scheduler.md

```diff
@@ -128,17 +138,17 @@ kube-scheduler has a default set of scheduling policies.
   This applies for both bound and unbound
   {{< glossary_tooltip text="PVCs" term_id="persistent-volume-claim" >}}.
 
-### Scoring
+#### Scoring
+
+The following priorities implement scoring:
```

Comment:
Is “priorities” the right word here?

Reply:
Yes. `Predicates` and `Priorities` are the names of the fields in the Policy to configure filtering and scoring, respectively.

Reply:
Is that name is from alpha code? If so, it's fixable and I might well (separately) suggest changing it: the English noun “priority” doesn't suggest _thing that determines part of an overall score for each of a set of items_. (Compare “filter” and “predicate”, which both suggest _thing that determines which items from a set are retained_).

Meanwhile, I wonder if there's a way to avoid introducing wording that might confuse readers.
I will try to suggest different wording.

Reply:
No, Policy is a v1 API :(
That's why our new v1alpha2 Profile API uses the term scoring instead.

Reply:
Ah, makes sense.

Reply:
Consider making priorities italic, to indicate that you're introducing a concept.